data/upgrades/device.json

```diff
@@ -118,5 +118,46 @@
     ],
     "cost": { "value": 3 },
     "hyperspace": true
+  },
+  {
+    "name": "DRK-1 Probe Droids",
+    "limited": 1,
+    "xws": "drk1probedroids",
+    "sides": [
+      {
+        "title": "DRK-1 Probe Droids",
+        "type": "Device",
+        "ability": "During the End Phase, you may spend 1 [Charge] to drop or launch 1 DRK-1 probe droid using a speed 3 template. This card's [Charge] cannopt be recovered.",
+        "image": "https://images-cdn.fantasyflightgames.com/filer_public/8d/db/8ddbbf77-aba5-41d2-8917-6866b43dfada/swz30_probe-droid.png",
+        "slots": ["Device"],
+        "charges": {
+          "value": 2,
+          "recovers": 0
+        },
+        "device": {
```

Comment:
This should've failed the build as `device` is not in the schema. Seems like the `sides` items don't have `"additionalProperties": false` set.

Reply:
All of the bombs and mines have `device` so I think we need to add it to the schema (or decide to encode information about the device that's dropped elsewhere).